src/pages/Admin/Admin.tsx

```diff
@@ -91,9 +92,9 @@ const VenueList: React.FC<VenueListProps> = ({
         </Link>
       </div>
       <ul className="page-container-adminsidebar-venueslist">
-        {venues.map((venue, index) => (
+        {relatedVenues.map((venue, index) => (
```

Comment:
The same comment for Admin NG applies here as well, except that these should be all venues that you are owner of, not only partymap.

Reply:
Replaced with `useAdministeredVenues()` that returns `administeredVenues` instead of `relatedVenues`  in f97606d53e7ceba1ab4ad0433711a96c16d5b7ea